src/App.js

```diff
@@ -1,26 +1,42 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React, { Component } from "react";
+import "./App.css";
+import { Switch, Route } from "react-router-dom";
+import LoginContainer from "./components/LoginContainer";
+import SignupContainer from "./components/SignupContainer";
+import { connect } from "react-redux";
```

Comment:
Do the proper sorting of yours deps.
Modules/absolute imports/relative import. Also, it good to see when it separated by a blank line.
You can add a rule to eslint for it and an editor will suggest to you the right order.

Reply:
https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/order.md
https://eslint.org/docs/rules/sort-imports
https://eslint.org/docs/rules/no-duplicate-imports